configure.ac

```diff
@@ -115,6 +119,24 @@ AC_SUBST(bootdir)
 AC_DEFINE_UNQUOTED(BOOT_DIRECTORY, "$bootdir",
                    [The normalised path to the boot/ESP mount])
 
+# The vendor name to be used in emitted boot entries
+AC_ARG_WITH([os-name], AS_HELP_STRING([--with-os-name],
+        [the OS name]), [osname=${withval}],
+        [osname="Clear Linux Software for Intel Architecture"])
```

Comment:
We could do. I mean ideally we should be using the filesystem's `os-release`, which would also allow a better pretty name & version tbh. So for Solus we see "Solus (Solus-lts..)" which is a bit unhelpful